run-tests.sh

```diff
@@ -44,7 +44,7 @@ fi
 python -m check_manifest
 python -m setup extract_messages --output-file /dev/null
 python -m sphinx.cmd.build -qnN docs docs/_build/html
-eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --cache ${CACHE:-redis} --env)"
```

Comment:
this should stay no? Otherwise, we cannot test search no?

Reply:
it was an addition of the commit i will revert. i thought to remove all what was added in the https://github.com/utnapischtim/invenio-accounts/commit/f9a8a8519b99abf0543886e230dfd73f870ad611

Reply:
I think we can keep that... Basically instead of `git revert <commit>` you can just push a new one with only removing the necessary parts

Reply:
if we add that to the tests then a container will be started which is not used. that consumes resources which are unnecessary 
since it was not there before in my opinion we should save resources